week4/example-telnetlib.py

```diff
@@ -0,0 +1,27 @@
+import getpass
+import telnetlib
+import time
+
+host = "10.0.15.101"
```

Comment:
Ip address is important, try to hide ip from resources which everyone can read it.